jsonnet/jsonnet.bzl

```
@@ -840,16 +840,16 @@ def jsonnet_repositories():
     """Adds the external dependencies needed for the Jsonnet rules."""
     http_archive(
         name = "jsonnet",
-        sha256 = "076b52edf888c01097010ad4299e3b2e7a72b60a41abbc65af364af1ed3c8dbe",
-        strip_prefix = "jsonnet-0.17.0",
+        sha256 = "85c240c4740f0c788c4d49f9c9c0942f5a2d1c2ae58b2c71068107bc80a3ced4",
+        strip_prefix = "jsonnet-0.18.0",
         urls = [
-            "https://github.com/google/jsonnet/archive/v0.17.0.tar.gz",
+            "https://github.com/google/jsonnet/archive/v0.18.0.tar.gz",
         ],
     )
 
     http_archive(
         name = "google_jsonnet_go",
-        sha256 = "4fd04d0c9e38572ef388d28ea6b1ac151b8a9a5026ff94e3a68bdbc18c4db38a",
-        strip_prefix = "go-jsonnet-0.17.0",
-        urls = ["https://github.com/google/go-jsonnet/archive/refs/tags/v0.17.0.tar.gz"],
+        sha256 = "20fdb3599c2325fb11a63860e7580705590faf732abf47ed144203715bd03a70",
+        strip_prefix = "go-jsonnet-0d78479d37eabd9451892dd02be2470145b4d4fa",
```

Small nit, can you add a comment about what version this is closes to?
```suggestion
        # go-jsonnet-0.18.0
        strip_prefix = "go-jsonnet-0d78479d37eabd9451892dd02be2470145b4d4fa",
```

---

I'd be happy to. Im not sure what the maintainers preference is. It appears in the past that all 3 of (jsonnet, go-jsonnet and rules_jsonnet) were versioned in unison. I personally think that makes sense for jsonnet and go-jsonnet, but not rules_jsonnet. However, I'm not a maintainer.

My suggestion for a comment would be:
`# https://github.com/google/go-jsonnet/commit/0d78479d37eabd9451892dd02be2470145b4d4fa`

I'll wait for the maintainers to provide their input.

---

It'd be nice if there was a `0.18.1` or something, but I find it nice to be able to recognize at a glance, generally what version I'm using when a commit hash is used as the pin. Again though, small nit, not a big deal. Thanks for working on this!